client/lib/signup/step-actions/index.js

```diff
@@ -1180,10 +1180,11 @@ export const buildUpgradeFunction = ( planProps, cartItem ) => {
 		themeSlugWithRepo,
 		goToNextStep,
 		submitSignupStep,
+		recordTracksEvent: rte,
 	} = planProps;
 
 	if ( cartItem ) {
-		recordTracksEvent( 'calypso_signup_plan_select', {
+		rte( 'calypso_signup_plan_select', {
```

Comment:
It's in general uncommon to abbreviate a function this way in calypso. When it happens, it's common to just use `this.props.recordTracksEvent()` to work around the linting rule, e.g. https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plans/current-plan/jetpack-checklist/index.tsx#L131. 

Let's use a more verbose form here :)